e2e-test-server/endtoendserver/server.go

```diff
@@ -0,0 +1,159 @@
+// Copyright 2019 OpenTelemetry Authors
```

Comment:
"OpenTelemetry Authors" attribution shouldn't be needed in any new files (unless they are copied in from OTel)

```suggestion
```

Reply:
Fixed